src/main/webapp/aboutus.jsp

```diff
@@ -41,8 +41,123 @@ limitations under the License.
                 });
       }
     </script>
+	 </script>
+  <title>Team 1's Location</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDTkxwf49JuT_CpGMbV4OHYMTQjp8BT0nE"></script>
+  <script>
+    let map;
+    /* Editable marker that displays when a user clicks in the map. */
+    let editMarker;
+    function createMap(){
+      map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: 38.5949, lng: -94.8923},
+        zoom: 4
+      });
+      // When the user clicks in the map, show a marker with a text box the user can edit.
+      map.addListener('click', (event) => {
+        createMarkerForEdit(event.latLng.lat(), event.latLng.lng());
+      });
+      fetchMarkers();
+	  addLandmark(map, 40.807048, -73.961038, 'Eleanor', 'Eleanor is currently in NYC') 
+      addLandmark(map, 33.063655, -96.771746, 'Antara', 'Antara is currently in Dallas') 
+      addLandmark(map, 42.026411, -93.643472, 'Kamsi', 'Kamsi is in Iowa');
+	  addLandmark(map, 37.422200, -122.083682, 'Kevin', 'Kevin is working in Mountain View')
+    }
+    /** Fetches markers from the backend and adds them to the map. */
+    function fetchMarkers(){
+      fetch('/markers').then((response) => {
+        return response.json();
+      }).then((markers) => {
+        markers.forEach((marker) => {
+         createMarkerForDisplay(marker.lat, marker.lng, marker.content)
+        });
+      });
+    }
+	  /** Adds a marker that shows an info window when clicked. */ 
```

Comment:
Nit: Can you fix the indentation here? Looks like a mix of spaces and tabs? (Google's style guide recommends using spaces.)